tutorials/git/README.md

```diff
@@ -0,0 +1,138 @@
+# git Tutorial
+
+### Outline
+
+The goal of this tutorial is to become familiar with some (of the vast) **git** functionality. It will cover the basics 
```

Comment:
It might be worth explaining what the difference between Git and GitHub is (you explain GitKraken later)

Reply:
excellent idea